docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+
+.. list-table:: 
+  :widths: 25 25 25 25
+  :header-rows: 1
+
+  * - **Id**
+    - **Source**
+    - **Destination**
+    - **Description**
+  * - **1**
+    - **User**
+    - **Wallet Instance**
+    - The User navigates the Verifier's website and clicks on the 'Login with IT Wallet' button
+  * - **2**
+    - **User**
+    - **Wallet Instance**
+    - The Authorization flow starts
+  * - **3-4**
+    - **Verifier (FE)**
+    - **Verifier (BE)**     
+    - Requests the generation of an Authorization Request which contains the Presentation Definition/Scope, Verifier metadata and information the wallet needs to generate the response.
+  * - **5**
+    - **Verifier (FE)**
+    - **-**     
+    - Inserts the reference to the previously generated object into a QR Code that is displayed on the screen
+  * - **6**
+    - **Verifier (FE)**
+    - **Wallet**
+    - QR Code is displayed ready to be scanned
+  * - **7**
+    - **Wallet**
+    - **-**
+    - Scans the QR Code
+  * - **8**
+    - **Wallet**
+    - **-**
+    - Extracts the Request URI from the payload of the QR Code
+  * - **9-10**
+    - **Wallet**
+    - **Verifier (BE)**          
+    - Requests the content of the Authorization Request by invoking the Request URI
+  * - **11**
+    - **Wallet**
+    - **-**     
+    - Verify the signature of the Request Object (JWT)
+  * - **12**
+    - **Wallet**
+    - **-**     
+    - Processes the content of the object and proceeds with the authentication
+  * - **13**
+    - **Wallet**
+    - **Verifier (BE)**     
+    - Generate an Authorization Response containing the VP Token
+  * - **14**
+    - **Verifier (BE)**
+    - **-**     
+    - Verify Authorization Response signature
+  * - **15**
+    - **Verifier (BE)**
+    - **-**     
+    - Process the content of the object, performs checks for integrity and tampering, and saves the Response to the persistence layer
+  * - **16**
+    - **Verifier (FE)**
+    - **Verifier (BE)**     
+    - Request a Session Cookie through polling, which will be available downstream after the checks performed in the previous step
+  * - **17**
+    - **Verifier (BE)**
+    - **-**     
+    - Consume the VP Token
+  * - **18**
+    - **Verifier (BE)**
+    - **-**     
+    - Requests the invalidation of the current session as the transaction has been successfully completed
+  * - **19**
+    - **Verifier (FE)**
+    - **User**     
+    - The user is successfully authenticated on the Verifier's website
+
+
+AuthorizationRequest Details
+----------------------------
+The AuthorizationRequest object is required by the Relying Party FrontEnd in order to show the QR Code. 
+The User will scan the QR Code using the Wallet granting the attributes to the RP.
+
+.. code-block:: javascript
+
+  POST /oid4vp HTTP/1.1
+  HOST: <VERIFIER_RELYING_PARTY_HOST>
+
+  HTTP/1.1 201 OK
+
+
+Below a non-normative response example
```

Comment:
What is the body of this POST? In particular, in the whole specification I don't see where the VP of the wallet instance comes into play!

Reply:
@TeamSog this section is not clear

We need a non normative example of the QR Code represented as SVG and another containing its payload

the payload should redirect the wallet to the request uri

Reply:
Pls check the authorization request detail section @grausof @peppelinux